pkg/apis/serving/v1beta1/configmap.go

```diff
@@ -65,17 +65,18 @@ type InferenceServicesConfig struct {
 
 // +kubebuilder:object:generate=false
 type IngressConfig struct {
-	IngressGateway          string  `json:"ingressGateway,omitempty"`
-	IngressServiceName      string  `json:"ingressService,omitempty"`
-	LocalGateway            string  `json:"localGateway,omitempty"`
-	LocalGatewayServiceName string  `json:"localGatewayService,omitempty"`
-	IngressDomain           string  `json:"ingressDomain,omitempty"`
-	IngressClassName        *string `json:"ingressClassName,omitempty"`
-	DomainTemplate          string  `json:"domainTemplate,omitempty"`
-	UrlScheme               string  `json:"urlScheme,omitempty"`
-	DisableIstioVirtualHost bool    `json:"disableIstioVirtualHost,omitempty"`
-	PathTemplate            string  `json:"pathTemplate,omitempty"`
-	DisableIngressCreation  bool    `json:"disableIngressCreation,omitempty"`
+	IngressGateway           string    `json:"ingressGateway,omitempty"`
+	IngressServiceName       string    `json:"ingressService,omitempty"`
+	LocalGateway             string    `json:"localGateway,omitempty"`
+	LocalGatewayServiceName  string    `json:"localGatewayService,omitempty"`
+	IngressDomain            string    `json:"ingressDomain,omitempty"`
+	IngressClassName         *string   `json:"ingressClassName,omitempty"`
+	AdditionalIngressDomains *[]string `json:"additionalIngressDomains,omitempty"`
```

Comment:
im not sure if this applies here but do you need to also update the charts/ file for helm releases like here https://github.com/kserve/kserve/blob/cefdc2ecff9764b318d975f1bfa31e71c6874362/charts/kserve-resources/templates/configmap.yaml#L491 or anything? I ran into this issue once, but im not sure if it applies here..

I just did a search for the variable above it `IngressClassName` to get an idea of all the files it is in https://github.com/search?q=repo%3Akserve%2Fkserve%20IngressClassName&type=code

Reply:
You were correct. I did not cover the chart.